src/comments.ts

```diff
@@ -81,6 +82,14 @@ and I'll merge this PR almost instantly. Thanks for helping out! :heart:
 ${otherOwners.length === 0 ? "" : `
 (${otherOwners.map(o => "@" + o).join(", ")}: you can do this too.)`}`});
 
+export const WaitUntilMergeIsOK = (user: string, abbrOid: string) => ({
+    // at most one reminder per update
+    tag: `wait-for-merge-offer-${abbrOid}`,
+    status: `Hi @${user}, I can't accept a merge request until the PR has a green CI and was appropriately reviewed. I will let you know once that happens.
+
+Thanks, and happy typing!`
```

Comment:
and this is cute.

Maybe we _could_ link to [the overview](https://github.com/DefinitelyTyped/dt-mergebot/blob/master/docs/dt-mergebot-lifecycle.png)

Reply:
Good idea, I made it point [here](https://github.com/DefinitelyTyped/DefinitelyTyped#make-a-pull-request) instead of directly at the image.